### PR TITLE
Docs: Update jaeger.md

### LINKED ISSUE #25170: Docs feedback: /features/datasources/jaeger.md
The link to "Loki documentation" in section "Linking Trace ID from logs" is broken.

### PR DESCRIPTION
**Pull request by oddlittlebird**
**What this PR does / why we need it**: Fixes a broken link

**Which issue(s) this PR fixes**:

Fixes #25170

**Special notes for your reviewer**:

### CODE REVIEW

**Review by ghost — CHANGES_REQUESTED** [review]
Hope this fixes link if not let me know

**Comment by oddlittlebird** [comment]
I do not see your suggestion.

File: docs/sources/features/datasources/jaeger.md [hunk 1]
```diff
@@ -44,4 +45,4 @@ Use the trace selector to pick particular trace from all traces logged in the ti
 
 ## Linking Trace ID from logs
 
-You can link to Jaeger trace from logs in Loki by configuring a derived field with internal link. See [Loki documentation]([Explore]({{< relref "./loki#derived-fields" >}})) for details.
+You can link to Jaeger trace from logs in Loki by configuring a derived field with internal link. See the [Derived fields]({{< relref "loki.md#derived-fields" >}}) section in the [Loki data source]({{< relref "loki.md" >}}) documentation for details.
```
Comment (ivanahuckova) [inline docs/sources/features/datasources/jaeger.md:48]:
```suggestion
You can link to Jaeger trace from logs in Loki by configuring a derived field with internal link. See the [Derived fields]({{< relref "./loki.md#derived-fields" >}}) section in the [Loki data source]({{< relref "./loki.md" >}}) documentation for details.
```

Comment (ivanahuckova) [reply docs/sources/features/datasources/jaeger.md:48]: I think that there need to be a full path to the file that we want to link. In the current version, we were just missing `.md`.

**Review by ivanahuckova — COMMENTED** [review]
I think that there need to be a full path to the file that we want to link. In the current version, we were just missing .md.

**Comment by cnk2182** [comment]
Sorry not to good at this stuff,but iam learning. If I understand correctly I would have to agree. How do I make that happen.

Sent from Outlook Mobile<https://aka.ms/blhgte>
________________________________

**Comment by oddlittlebird** [comment]
@ivanahuckova , why is the enterprise frontend test failing?

**Comment by cnk2182** [comment]
To say the least

Sent from Outlook Mobile<https://aka.ms/blhgte>
________________________________

**Comment by ivanahuckova** [comment]
@oddlittlebird not sure - probably some change that happened between branching off and now.  But I merged the current master and that should fix the problem. :)

**Review by ivanahuckova — APPROVED** [review]

**Pull request merged**